virtual_keyboard/index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <main>
+    <section class="first-row row">
+      <button class="backtick dark-blue"><span>~</span> `</button>
```

Comment:
use meaningful name of class instead dark-blue. I discribe what I mean in css file. in comment for .dark-blue